internal/querycoord/task.go

```diff
@@ -688,6 +688,19 @@ func (rct *releaseCollectionTask) execute(ctx context.Context) error {
 			return err
 		}
 
+		collInfo, err := rct.meta.getCollectionInfoByID(collectionID)
```

Comment:
this is weird, invalidateCollectionMetaCache should use collectionId rather than collection name since it is an internal call.

proxypb.InvalidateCollMetaCacheRequest should accepect either collection name or collection id

Reply:
fixed, now all the meta cache with specified collectionID will be removed